contrib/compose/README.md

```diff
@@ -0,0 +1,23 @@
+# Docker Compose V1 + V2 registry
+
+This compose configuration will setup a v1 and v2 registry behind an nginx
+proxy. By default the combined registry may be accessed at localhost:5000.
+This registry does not support pushing images to v2 and pull from v1. Clients
+from before 1.6 will be configured to use the v1 registry, and newer clients
+will use the v2 registry.
+
+## Prerequisites
+Install [docker-compose](https://github.com/docker/compose)
+
+## How to run
+~~~
```

Comment:
Not sure about this repo, but in the docker repo, either three back-ticks or indenting with 4 spaces is used for code blocks. Just mentioning, you may consider that to keep it consistent.